package.json

```diff
@@ -40,7 +40,7 @@
     "homepage": "https://github.com/supertokens/supertokens-web-js#readme",
     "dependencies": {
         "supertokens-js-override": "0.0.4",
-        "supertokens-website": "^12.0.0"
+        "supertokens-website": "github:supertokens/supertokens-website#next-breaking-change"
```

Comment:
this needs to change to the right version. Keeping this PR open until this is done.